redaxo/src/addons/phpmailer/pages/config.php

```diff
@@ -51,13 +52,24 @@
     $sel_mailer->addOption($type, $type);
 }
 
+
+$sel_security_mode = new rex_select();
+$sel_security_mode->setId('security_mode');
+$sel_security_mode->setName('settings[security_mode]');
+$sel_security_mode->setSize(1);
+$sel_security_mode->setAttribute('class', 'form-control selectpicker');
+$sel_security_mode->setSelected($this->getConfig('security_mode'));
```

Comment:
Sollte der config key besser auch „autotls“ heißen? Oder warum hast du bewusst einen komplett anderen mey gewählt?

Reply:
autotls ist die ursprüngliche Bezeichnung - ich ändere das

Reply:
das könnte man in einem separaten PR noch fixen.. ist vermutlich vergessen worden